centreon/tests/e2e/features/Migration-scripts/02-media.feature

```diff
@@ -0,0 +1,69 @@
+@ignore
+@REQ_MON-24294
+
+Feature: Migration of medias from a source platform to a target platform
+
+  Background:
+    Given an admin user who wants to migrate medias from a source platform to a target platform
+    And the user has access to both the source and target platforms
+    And another non-admin user has the necessary rights to manage medias
+    And the non-admin user has access to both the source and target platforms
+
+  Scenario: Medias creation on the source platform
+    Given a non-admin user logged in on the source platform
+    When the user adds a new media
+    Then the media is added and listed on the source platform
+
+  Scenario: Execution of the migration script with the complete command
+    Given an admin user logged in on the source platform on the terminal
+    When the user runs the following command in the terminal:
+    """
+        php /usr/share/centreon/bin/migration media:all {target_url}
+    """
+    Then a command line is displayed asking for the API token of the target platform
```

Comment:
Since this is run on terminal, shouldn t it be something like "Then the user is asked for the API token of the target platform" (I may be wrong, but I think cypress can not assert the displayed output of the terminal, but only the raw output)